components/info/component.go

```diff
@@ -36,6 +46,19 @@ const (
 	StateKeyMacAddress    = "mac_address"
 	StateKeyPackages      = "packages"
 
+	StateKeyGPUdPID = "gpud_pid"
+
+	StateKeyGPUdUsageFileDescriptors = "gpud_usage_file_descriptors"
+
+	StateKeyGPUdUsageMemoryInBytes   = "gpud_usage_memory_in_bytes"
+	StateKeyGPUdUsageMemoryHumanized = "gpud_usage_memory_humanized"
```

Comment:
overall lgtm. I'm curious if we need these keys with the suffix "humanized"

Reply:
Useful for user-facing UI :)